releng/website/downloads/latest-S/compositeArtifacts.xml

```diff
@@ -2,9 +2,9 @@
 <repository name='Stable Orbit Aggregation'
     type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
   <properties size='1'>
-    <property name='p2.timestamp' value='1690422378727'/>
+    <property name='p2.timestamp' value='1694197952781'/>
   </properties>
   <children size='1'>
-    <child location='https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09'/>
+    <child location='https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/milestone/latest'/>
```

Comment:
@merks is this best URL - I had previously pointed latest-S to the versioned orbit-aggregation, but I don't really want to make updates in the future and AFAICT milestone/latest looks right.

Reply:
milestone latest will eventually be m1 for 2023-12 / 4.30 so that seems good/right

Reply:
Brilliant - Thanks Ed.